packages/kit/src/runtime/app/server/index.js

```diff
@@ -0,0 +1,43 @@
+import { read_asset, manifest } from '__sveltekit/server';
+
+/**
+ * TODO docs
+ * @param {string} file
+ * @returns {Response}
+ */
+export function readAsset(file) {
```

Comment:
I'm not sure about this name. `assets` is used to mean the contents of `static` as well as the build output. Not sure what would be better

Reply:
maybe just `read`